changelogutils/validator.go

```diff
@@ -94,6 +101,9 @@ type changelogValidator struct {
 	reader ChangelogReader
 	client githubutils.RepoClient
 	code   vfsutils.MountedRepo
+	// list of arbitrary labels whos order is used to tie-break tag comparisons between
+	// versions with different labels. Labels ordered earlier are greater.
+	labelOrder []string
```

Comment:
I'm scared to hardcode any sort of arbitrary label ordering into go-utils and leave this as an implementation detail to changelog-bot.